### ISSUE TITLE
PowerSync.init() doesn't return on Windows

### ISSUE THREAD

**Issue by hut36** [powersync-ja/powersync-js#104]
The demo repo is at https://github.com/hut36/tauri-powersync.git. This is a tauri project, which works fine on macOS.

The code in question is:

```
	async function setupPowerSync() {
		let result

		console.log('before powersync init')

		await PowerSync.init();  // <--- never returns on Windows.

		console.log('after powersync init')

		result = await PowerSync.execute(`insert into lists (id, name) values (uuid(), 'list0')`)
		console.log('insert result: ', result)

		result = await PowerSync.execute('select * from lists')
		console.log('select result: ', result)
	}
```

**Comment by benitav** [comment]
Hey @hut36 - thanks for this. We don't directly support Tauri yet. Using the web SDK (with IndexedDB storage) might work, but we haven't actively tested this so there might be unexpected issues (such as this one, potentially). If we do get a chance we'll investigate.

**Comment by hut36** [comment]
I've narrowed it down to `SharedWorker`. On Windows multiTabs is enabled, so `SharedWorker` is used. However, the `onconnect` handle in file `SharedWASQLiteDB.worker.ts` is not triggered, the program is blocked on the initialization of DB.

The webkit used by tauri on my macOS doesn't support `SharedWorker` thus multiTabs is disabled by default, thus the problem is not observed.

I set up a minimal tauri project to test with `SharedWorker`, and it didn't work. So the problem is probably related to tauri and `SharedWorker`.
